boltstub/tests/_common.py

```diff
@@ -38,6 +38,7 @@ def cycle_zip(*args):
     (3,),
     (4, 1), (4, 2), (4, 3), (4, 4),
     (5, 0), (5, 1), (5, 2), (5, 3), (5, 4), (5, 5), (5, 6), (5, 7),
```

Comment:
```suggestion
    (5, 0), (5, 1), (5, 2), (5, 3), (5, 4), (5, 5), (5, 6), (5, 7), (5, 8),
```
Looks like we're missing 5.8 here?